python/nano/src/bigdl/nano/deps/ipex/ipex_inference_model.py

```diff
@@ -117,18 +157,22 @@ def __getattr__(self, name: str):
     @property
     def status(self):
         status = super().status
-        status.update({"use_ipex": self.use_ipex,
-                       "use_jit": self.use_jit,
-                       "channels_last": self.channels_last,
-                       "checkpoint": "ckpt.pth",
-                       "thread_num": self.thread_num,
-                       "jit_strict": self.jit_strict})
+        status.update(
+            {
+                "use_ipex": self.use_ipex,
+                "use_jit": self.use_jit,
+                "channels_last": self.channels_last,
+                "checkpoint": "ckpt.pth",
+                "thread_num": self.thread_num,
+                "jit_strict": self.jit_strict,
```

Comment:
should we save the `self.channels_last_available` here? Since we do want it be recoverred after users calling save/load